kolibri/deployment/default/settings/base.py

```diff
@@ -190,6 +190,21 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+# File Storage Backend
+# https://docs.djangoproject.com/en/3.2/ref/files/storage/
+
+if not os.environ.get("DEFAULT_FILE_STORAGE"):
+    if conf.OPTIONS["FileStorage"]["STORAGE_BACKEND"] == "gcs":
+        # Initialize GS_CREDENTIALS as a proper google.auth.credentials.Credentials object
+        import google.auth
+
+        GS_CREDENTIALS, _ = google.auth.default()
```

Comment:
@DXCanas this is where I ended up putting the code we need to generate the credentials object. @rtibbles is there somewhere better to put this?

Reply:
Could we initialize this during the options parsing as a second step in validation? That way we could fail much earlier if the credentials are incorrectly configured.